purchases/src/main/kotlin/com/revenuecat/purchases/Purchases.kt

```diff
@@ -209,8 +208,7 @@ class Purchases internal constructor(
             fetchAndCacheOfferings(identityManager.currentAppUserID, appInBackground = false)
             log(LogIntent.RC_SUCCESS, OfferingStrings.OFFERINGS_UPDATED_FROM_NETWORK)
         }
-        // Offline entitlements: Commenting out for now until backend is ready
-        // offlineEntitlementsManager.updateProductEntitlementMappingCacheIfStale()
+        offlineEntitlementsManager.updateProductEntitlementMappingCacheIfStale()
```

Comment:
This is fine for now, since we have an internal feature flag in `AppConfig` that disables it.